etrago/appl.py

```diff
@@ -445,13 +453,24 @@ def run_etrago(args, json_path):
             as well and included in number of clusters specified through
             ``'n_clusters_AC'``.
             Default: False.
+        * "interest_area": False, list, string
+            Area of especial interest that will be not clustered. It is by
+            default set to false. When an interest_area is provided, the given
+            value for n_clusters_AC will mean the total of AC buses outside the
+            area.The area can be provided in two ways: list of
+            nuts names e.G. ["Cuxhaven", "Bremerhaven", "Bremen"] or a string
+            with a path to a shape file (.shp).
+            Default: False.
+        * "cluster_interest_area": False, int
```

Comment:
I would rename this to "n_cluster_interest_area" to be more consistent with the other parameters.